substrate/frame/contracts/src/wasm/prepare.rs

```diff
@@ -312,7 +315,8 @@ pub mod benchmarking {
 		owner: AccountIdOf<T>,
 	) -> Result<WasmBlob<T>, DispatchError> {
 		let determinism = Determinism::Enforced;
-		let contract_module = LoadedModule::new::<T>(&code, determinism, None)?;
+		let compilation_mode = CompilationMode::Lazy;
+		let contract_module = LoadedModule::new::<T>(&code, determinism, None, compilation_mode)?;
```

Comment:
Can be eager, too. Shouldn't make any difference, though. As it is only used in benchmarking setup code.

Reply:
done